fusillade/api/roles/__init__.py

```diff
@@ -1,24 +1,32 @@
 from flask import request, make_response, jsonify
 
 from fusillade import Role, directory
+from fusillade.utils.authorize import assert_authorized
 
 
-def put_new_role():
+def put_new_role(user):
+    assert_authorized(user, ['fus:PutRole'], ['arn:hca:fus:*:*:role'])
     json_body = request.json
     Role.create(directory, json_body['name'], statement=json_body.get('policy'))
-    return make_response('', 201)
+    return make_response(f"New role {json_body['name']} created.", 201)
 
 
 def get_roles():
     pass
 
 
-def get_role(role_id):
+def get_role(user, role_id):
```

Comment:
The method parameters in this file are missing type annotations.

Reply:
will add